client/test/unit/specs/ActionButtons.spec.js

```diff
@@ -141,6 +141,7 @@ describe('ActionButtons.vue', () => {
 
     it('hides decision and hold buttons for IN PROGRESS NR of another examiner', () => {
       expect(vm.$el.querySelector('#examine-decide-button')).toBeNull()
+      console.log(userId)
```

Comment:
FYI this console log causes the test to error (consistently for me). When I removed it the test passed, so the test and functionality itself seem fine but fails when run.

Reply:
This has been corrected.  All test ran properly for me.